docs/doxygen-user/portable_case.dox

```diff
@@ -0,0 +1,48 @@
+/*! \page portable_case_page Portable Cases
+
+\section portable_case_overview Overview
+
+A portable case is a partial copy of a normal Autopsy case that can be opened from anywhere. The general use case is as follows:
+<ol>
+<li>Alice is analyzing one or more data sources using Autopsy. She tags files and results that are of particular interest.
+<li>Alice wants to share her findings with Bob but is unable to send him the original data sources.
+<li>Alice creates a portable case which will contain only her tagged files and results, plus any files associated with those results, and sends it to Bob.
+<li>Bob can open the portable case in Autopsy and view all files and results Alice tagged, and run any of the normal Autopsy features.
+</ol>
+
+For example, Alice's original case could look like this:
+
+\image html portable_case_original_version.png
+
+The the portable version could like this:
+
+\image html portable_case_portable_version.png
+
+Alice only tagged eight files and results, so most of the original content is no longer in the case. Some of the data sources had no tagged items so they're not included at all. The file structure of any tagged files is preserved - you can see that the tagged image in the screenshot is still in the same location, but the non-tagged files are gone. Note that although the original images (such as "image1.vhd") appear in the tree, their contents are not included in the portable case.
+
+\section portable_case_creation Creating a Portable Case
+
+First you'll want to make sure that all the files and results you want included in the portable case are tagged - see the \ref tagging_page page for more details. 
+You can see what tags you've added in the \ref tree_viewer_page.
+
+\image html portable_case_tags.png
+
+Portable cases are created through the \ref reporting_page feature. The dialog will display a list of all tags that are in use in the current case and you can choose which ones you would like to include. At the bottom you can select the output folder for the new case. By default it will be placed in the "Reports" folder in the current case.
```

Comment:
I think it would be helpful for tying things together to say something like "the Generate Reports dialog" (or whatever it's title is).